lib/lita-reviewme/github.rb

```diff
@@ -0,0 +1,66 @@
+require 'delegate'
+
+module Lita
+  module Reviewme
+    class Github
```

Comment:
Did you consider testing this object in isolation? It stands on it's own, so you might learn something about its design by isolating it in test.